File: 백준/Bronze/28431. 양말 짝 맞추기/양말 짝 맞추기.cs
using System;

class Program
{
    static void Main()
    {
        int[] a = new int[5];
        
        for (int i = 0; i < 5; i++)
        {
            a[i] = int.Parse(Console.ReadLine());
        }

        int result = 0;
        for (int i = 0; i < 5; i++)
        {
            int count = 0;
            for (int j = 0; j < 5; j++)
            {
                if (a[i] == a[j])
                    count++;
            }
            if (count % 2 == 1)
            {
                result = a[i];
                break;
            }
        }

        Console.WriteLine(result);
    }
}

File: 백준/Bronze/28431. 양말 짝 맞추기/양말 짝 맞추기.py
a = [int(input()) for _ in range(5)]

result = None
for i in range(5):
    count = 0
    for j in range(5):
        if a[i] == a[j]:
            count += 1
    if count % 2 == 1:
        result = a[i]
        break

print(result)
